### DISCUSSION TITLE
Tags for games

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#696] [Ideas]
Developers could create tags or use existing ones to mark games. Tags could bring subgenres, be something similiar to IMDb plot keywords or technical details. For example (about Phantasy Star): party-based, science fantasy, revenge, dungeon crawl, female protagonist, cat protagonist, space travel, vehicle travel, robot, medusa, cat, light saber, dragon, dome, prison, FM compatible, world map, turn based combat, first person view combat, random encounters. 
Those tags could be grouped for more convenient use. For example:
- game genre (dungeon crawl)
- plot genre (science fantasy)
- plot element (revenge)
- playable character (party-based, female protagonist, cat protagonist)
- exploration (space travel, vehicle travel)
- creature (robot, medusa, cat, dragon)
- environment and place (dome, prison, world map)
- items (light saber)
- music and sound (FM compatible)
- combat (turn based combat, first person combat, random encounters)

More popular details like e.g. magic, cave, knight, male protagonis, forest (especially base elements of a genre) should be skipped. Similar Games could be selected authomatically, depending on matching tags and game lables

**Comment by kdecks** [comment]
Noteworthy, this is somewhat addressed with linking category pages (aka ~~hubs~~) http://retroachievements.org/game/5771
